TA-WellArchitected/tawa-optimization-starter/README.md

```diff
@@ -0,0 +1,117 @@
+# Trusted Advisor - Well-Architected Optimization - starter kit
+
+Author / Contributor
+* Stephen Salim
+* Carlos Perez
+
+
+![TA-WA-Optimizer-HighArch.png](./static/images/TA-WA-Optimizer-HighArch.png)
```

Comment:
This should be consistent to TA-WA, not TA-WellArchitected like the title"

Reply:
This is now changed, the title is now TA-WA Optimization - starter